test/e2e/browserslist-extends/index.test.ts

```diff
@@ -0,0 +1,38 @@
+import { createNext } from 'e2e-utils'
+import { NextInstance } from 'test/lib/next-modes/base'
+import { renderViaHTTP } from 'next-test-utils'
+
+describe('browserslist-extends', () => {
+  let next: NextInstance
+
+  beforeAll(async () => {
+    next = await createNext({
+      files: {
+        'pages/index.js': `
+          import styles from './index.module.css'
+          
+          export default function Page() { 
+            return <p className={styles.hello}>hello world</p>
+          } 
+        `,
```

Comment:
Is there anything specifix to this code that browser list is interacting with? Or is it more of a hello world?

Reply:
Mainly just a hello world, it failed from just reading the `browserslist` config in `package.json` before this fix.